src/content/reference/react/createFactory.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: createFactory
+Judul : createFactory
```

Comment:
Is this valid?
cc @mazipan @resir014 @r17x @mhaidarhanif

Reply:
@milhamm No, keep it. That's required variable for the docs generation.

Reply:
```suggestion
title: createFactory
```